code/game/atoms.dm

```diff
@@ -132,6 +132,9 @@
 /atom/proc/is_open_container()
 	return flags & OPENCONTAINER
 
+atom/proc/is_transparent()
+	return flags & TRANSPARENT
+

```

Comment:
Coiax, I am unsure how much me just changing these two procs would help, at all. Since I think we are at around 25k procs(the main procs and their children). Such a thing would probably have to be a whole PR of itself, focussing on more than just 1 or 2 procs.